pkg/runtime.v2/framework/plugins/jobset/builder.go

```diff
@@ -51,6 +52,90 @@ func NewBuilder(objectKey client.ObjectKey, jobSetTemplateSpec kubeflowv2.JobSet
 	}
 }
 
+// mergeInitializerEnvs merges the TrainJob and Runtime Pod envs.
+func mergeInitializerEnvs(storageUri *string, trainJobEnvs, containerEnv []corev1.EnvVar) []corev1.EnvVar {
```

Comment:
```suggestion
func mergeInitializerEnvs(storageUri string, trainJobEnvs, containerEnv []corev1.EnvVar) []corev1.EnvVar {
```

It seems that this pointer is less valuable.

Reply:
Actually, I use this pointer here to verify if StorageUri is set: https://github.com/kubeflow/training-operator/blob/e9f017c62f724cc23535fc06f1f1c0ddc00e054e/pkg/runtime.v2/framework/plugins/jobset/builder.go#L60

Does it make sense ?

Reply:
Ah, I was just missed the line. That makes sense. Thanks.